object-manipulation-3/main.js

```diff
@@ -0,0 +1,116 @@
+  } else if (scoresInOrder[m] === 'Jack' || scoresInOrder[m] === 'Queen' || scoresInOrder[m] === 'King') {
+    scoresValue.push(10);
+  } else if (scoresInOrder[m] === 2) {
+    scoresValue.push(2);
+  } else if (scoresInOrder[m] === 3) {
+    scoresValue.push(3);
+  } else if (scoresInOrder[m] === 4) {
+    scoresValue.push(4);
+  } else if (scoresInOrder[m] === 5) {
+    scoresValue.push(5);
+  } else if (scoresInOrder[m] === 6) {
+    scoresValue.push(6);
+  } else if (scoresInOrder[m] === 7) {
+    scoresValue.push(7);
+  } else if (scoresInOrder[m] === 8) {
+    scoresValue.push(8);
+  } else {
+    scoresValue.push(9);
+  }
+}
+var playerByIndex = [];
+for (var j = 0; j < scoresValue.length; j++) {
+  playerByIndex.push(scoresValue[j] + scoresValue[j++]);
+}
+console.log('Players 1-4:', playerByIndex);
+var final1 = playerByIndex[0];
+var final2 = playerByIndex[1];
+var final3 = playerByIndex[2];
+var final4 = playerByIndex[3];
+if (final1 > final2 && final1 > final3 && final1 > final4) {
+  console.log('%cPlayer1 Wins!', 'color: cyan; font-size: 15px');
+} else if (final2 > final1 && final2 > final3 && final2 > final4) {
+  console.log('%cPlayer2 Wins!', 'color: cyan; font-size: 15px');
+} else if (final3 > final1 && final3 > final2 && final3 > final4) {
+  console.log('%cPlayer3 Wins!', 'color: cyan; font-size: 15px');
+} else if (final4 > final1 && final4 > final2 && final4 > final3) {
+  console.log('%cPlayer4 Wins!', 'color: cyan; font-size: 15px');
+} else if (final1 === final2 && final1 === final3 && final1 === final4) {
+  console.log('%cEveryone Tied!', 'color: cyan; font-size: 15px');
+} else if ((final1 && final2) > (final3 && final4)) {
+  console.log('Player1 and Player2 Tied!');
+  tieBreaker('Player 1', 'Player2');
+} else if ((final1 && final3) > (final2 && final4)) {
+  console.log('Player1 and Player3 Tied!');
+  tieBreaker('Player 1', 'Player3');
+} else if ((final1 && final4) > (final2 && final3)) {
+  console.log('Player1 and Player4 Tied!');
+  tieBreaker('Player 1', 'Player4');
+} else if ((final2 && final3) > (final1 && final4)) {
+  console.log('Player2 and Player3 Tied!');
+  tieBreaker('Player 2', 'Player3');
+} else if ((final2 && final4) > (final1 && final3)) {
+  console.log('Player2 and Player4 Tied');
+  tieBreaker('Player 2', 'Player4');
+} else if ((final3 && final4) > (final1 && final2)) {
+  console.log('Player3 and Player4 Tied!');
+  tieBreaker('Player 3', 'Player4');
+}
+function tieBreaker(person1, person2) {
+  console.log(person1 + ' and ' + person2 + ' pull another card!');
+  var tie1 = _.random(2, 11);
+  var tie2 = _.random(2, 11);
+  if (tie1 > tie2) {
+    console.log(person1 + ' wins! Their card is worth ' + tie1 + ' points');
+    console.log(person2 + ' pulled a card worth ' + tie2 + ' points.');
+  } else {
+    console.log(person2 + ' wins! Their card is worth ' + tie2 + ' points');
+    console.log(person1 + ' pulled a card worth ' + tie1 + ' points.');
+
+  }
+}
```

Comment:
This shouldn't require a separate process from what we've already done to determine the winner. If there is a tie, you should just start a new game in between those players who are tied and determine the winner for that game. If there is a tie in that game, start another game, then repeat this process as many times as necessary to get to a single winner.